recipes/edward/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "edward" %}
+{% set version = "1.3.3" %}
+{% set sha256 = "104d58321c5040235b3039ae3215c3c7881073e6aa88bb0b8ca1141ca87c4891" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
+    - six >=1.10.0
+  run:
+    - python
+    - numpy >=1.7
+    - six >=1.10.0
+
+test:
+  imports:
+    - edward
+    - edward.criticisms
+    - edward.inferences
+    - edward.inferences.conjugacy
+    - edward.models
+    - edward.util
+
+about:
+  home: http://edwardlib.org
+  license: Apache
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+
+  description: |
+    Edward is a Python library for probabilistic modelling,
+    inference and criticism. Edward is built on top of Tensorflow.
+  doc_url: http://edwardlib.org/
+  dev_url: https://github.com/blei-lab/edward
+
+extra:
+  recipe-maintainers:
+    - souravsingh
+    - dustinvtran
```

Comment:
@dustinvtran -- are you ok being added as a co-maintainer of the `edward` recipe on conda-forge?